src/pages/api/sepana/update.page.ts

```diff
@@ -1,132 +1,80 @@
 import { isBigNumberish } from '@ethersproject/bignumber/lib/bignumber'
-import { readProvider } from 'constants/readProvider'
-import { infuraApi } from 'lib/infura/ipfs'
 import { queryAll, writeSepanaRecords } from 'lib/sepana/api'
 import { sepanaLog } from 'lib/sepana/log'
 import { Json } from 'models/json'
-import { AnyProjectMetadata } from 'models/projectMetadata'
 import { SepanaProject } from 'models/sepana'
-import { Project } from 'models/subgraph-entities/vX/project'
 import { NextApiHandler } from 'next'
 import { formatError } from 'utils/format/formatError'
 import { formatWad } from 'utils/format/formatNumber'
 import { querySubgraphExhaustiveRaw } from 'utils/graph'
-import { openIpfsUrl } from 'utils/ipfs'
-
-type ProjectKey =
-  | 'id'
-  | 'projectId'
-  | 'pv'
-  | 'handle'
-  | 'metadataUri'
-  | 'currentBalance'
-  | 'totalPaid'
-  | 'createdAt'
-  | 'trendingScore'
-  | 'deployer'
-
-const projectKeys: ProjectKey[] = [
-  'id',
-  'projectId',
-  'pv',
-  'handle',
-  'metadataUri',
-  'currentBalance',
-  'totalPaid',
-  'createdAt',
-  'trendingScore',
-  'deployer',
-]
+import {
+  getChangedSubgraphProjects,
+  sgSepanaCompareKeys,
+  tryResolveMetadata,
+} from 'utils/sepana'
 
 // Synchronizes the Sepana engine with the latest Juicebox Subgraph/IPFS data
 const handler: NextApiHandler = async (req, res) => {
   try {
     // This flag will let us know we should retry resolving IPFS data for projects that are missing it
-    const retryIPFS = req.method === 'POST' && req.body['retryIPFS'] === true
-
-    // Load all projects from Sepana
-    const sepanaProjects = (await queryAll<Json<SepanaProject>>()).data.hits
-      .hits
+    const retryIpfs = req.body['retryIPFS'] === true
```

Comment:
**suggestion (non0blocking)** I guess this is equiv to : 
```suggestion
    const retryIpfs = req.body.retryIPFS === true
```
Or
```suggestion
    const retryIpfs = Boolean(req.body.retryIPFS)
```

Reply:
second would resolve to true if the `retryIPFS` param was anything truthy, so would rather leave the explicit `=== true`. first suggestion makes sense!